# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../inkscape/inkscape-0.46-gtk2.13.3.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

gtkctree.h and gtkclist.h include each other, the order of inclusion
is important in gtk-2.13.3
https://bugs.launchpad.net/inkscape/+bug/246868

Lubomir Rintel <lkundrak@v3.sk>

--- inkscape-0.46+devel.orig/src/widgets/sp-xmlview-attr-list.h	2008-06-24 13:38:07.000000000 +0200
+++ inkscape-0.46+devel/src/widgets/sp-xmlview-attr-list.h	2008-07-05 13:03:50.000000000 +0200
@@ -13,6 +13,7 @@
  */
 
 #include <stdio.h>
+#include <gtk/gtkctree.h>
 #include <gtk/gtkclist.h>
 #include "../xml/repr.h"
 
